Lesson8/src/RecordString.java

```diff
@@ -0,0 +1,32 @@
+import java.io.FileOutputStream;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Scanner;
+
+public class RecordString {
+    private static final String RESOURCE_DIR = "Lesson8/resources/";
+    private static final String OUTPUT_FILE = RESOURCE_DIR + "output.txt";
+
+    public void recordReversString() {
+        Scanner scanner = new Scanner(System.in);
+        System.out.println("Could You enter line, please");
+        String string = scanner.nextLine();
+        char[] writer = new RecordString().revers(string);
+
+        try (FileWriter fileWriter = new FileWriter(OUTPUT_FILE, true)) {
```

Comment:
I don't think you want to have append flag as true here. Did you do it on purpose?